src/navigation/reroute.js

```diff
@@ -91,7 +91,7 @@ export function reroute(pendingPromises = [], eventArguments) {
   }
 
   function performAppChanges() {
-    return Promise.resolve().then(() => {
+    return Promise.resolve().then(async () => {
```

Comment:
We don't use async/await in single-spa, as we supports IE11 and compiling async/await requires babel's regenerator runtime, which is a fair number of bytes. You can see in https://github.com/single-spa/single-spa/pull/733#issuecomment-775119086 that adding this async function added ~900 bytes to the minified output of IE11-compatible bundles while only adding ~200 bytes to the corresponding es2015 bundles.

Please remove this in favor of using promises without the async/await sugar.